src/components/molecules/FormFields/Terms.tsx

```diff
@@ -23,7 +23,7 @@ export default function Terms(props: InputProps): ReactElement {
     <>
       <div
         className={styles.terms}
-        dangerouslySetInnerHTML={{ __html: data.terms.html }}
+        // dangerouslySetInnerHTML={{ __html: data.terms.html }}
```

Comment:
I think this is a good working adjustment. We can remove the comments and merge this. :+1: